### ISSUE TITLE
When to supply the service manifest?

### ISSUE THREAD

**Issue by gitonthescene** [reconciliation-api/census#10]
From [this example](https://github.com/mikejs/reconcile-demo/blob/master/reconcile.py) I understand that all interaction comes from a single endpoint and that the service manifest is to be delivered from the end point if it's not a query.  I think it might be worth making that explicit in the API.  If this is reason for leaving this ambiguous, then I think an explanation of why would be helpful.  Should the service manifest always be accessible with a GET request?

**Comment by wetneb** [comment]
Yes the manifest should be served when none of `query`, `queries` and `extend` are supplied. Feel free to open a PR to clarify this!

**Comment by gitonthescene** [comment]
I don't feel comfortable explaining an intent I don't fully understand.  There's no clear mention of the `query` parameter either.

**Comment by gitonthescene** [comment]
Also, the callback code in the example seems necessary with OpenRefine version 3.4.1.  I'm not sure why.  This seems like it probably should also have an explanation in the API spec.

**Comment by wetneb** [comment]
The "callback code" is what we call JSONP support.

Unless there is a bug, OpenRefine 3.4.1 should first try to contact a service via CORS and if that fails, it falls back on JSONP, to be compatible with previous versions of the spec.

In the current spec, service MUST implement CORS, so you should do that before adding support for JSONP.
